### DISCUSSION TITLE
25.0.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1923] [General]
## Releases Notes for 25.0.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/draw.io-25.0.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/draw.io-25.0.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/draw.io-universal-25.0.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/drawio-amd64-25.0.2.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/drawio-x86_64-25.0.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.2/drawio-x86_64-25.0.2.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- #1922 
- Updates to draw.io core 25.0.2
- Uses electron 32.2.5
- Updates to [draw.io core 25.0.2](https://github.com/jgraph/drawio/blob/v25.0.2/ChangeLog). All changes from 25.0.2 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v25.0.2'>25.0.2</a>.</em>

**Comment by dpa2007** [comment]
Greetings all,
is there a File_*hashes.txt hash txt there, for the latest version 25.02?
I didn't see any...
thanks in advance
